src/jobs/index.ts

```diff
@@ -74,39 +86,44 @@ export async function handleJobRoute(
   tx.finish();
 }
 
-export const opts = {
-  schema: {
-    body: {
-      type: 'object',
-      required: ['message'],
-      properties: {
-        message: {
-          type: 'object',
-          required: ['data'],
-          properties: {
-            data: {
-              type: 'string',
-            },
-          },
-        },
-      },
-    },
-  },
-};
+// Error handling wrapper function for Cron Jobs
+// Additionally handles Auth from Cloud Scheduler
+export async function handleCronJobs(
+  handler,
+  request: FastifyRequest,
+  reply: FastifyReply
+) {
+  const verified = await handleCronAuth(request, reply);
+  if (!verified) {
+    return; // Reply is already sent, so no need to re-send
+  }
+  const tx = Sentry.startTransaction({
+    op: 'webhooks',
+    name: 'jobs.jobsHandler',
+  });
+
+  reply.code(204);
+  reply.send(); // Respond early to not block the webhook sender
+  await handler();
+  tx.finish();
+}
 
 // Function that creates a sub fastify server for job webhooks
 export async function routeJobs(server: Fastify, _options): Promise<void> {
   server.post('/stale-triage-notifier', (request, reply) =>
-    handleJobRoute(notifyProductOwnersForUntriagedIssues, request, reply)
+    handleGithubJobs(notifyProductOwnersForUntriagedIssues, request, reply)
   );
   server.post('/stale-bot', (request, reply) =>
-    handleJobRoute(triggerStaleBot, request, reply)
+    handleGithubJobs(triggerStaleBot, request, reply)
   );
   server.post('/slack-scores', (request, reply) =>
-    handleJobRoute(triggerSlackScores, request, reply)
+    handleGithubJobs(triggerSlackScores, request, reply)
   );
   server.post('/gocd-paused-pipeline-bot', (request, reply) =>
-    handleJobRoute(triggerPausedPipelineBot, request, reply)
+    handleGithubJobs(triggerPausedPipelineBot, request, reply)
```

Comment:
While this did use the same handler previously, it would be nice to decouple this job from the GitHub logic since it is unrelated to it and doesn't use the org passed in. The only reason it was written that way originally was for convenience as there was only one mechanism for handling cron jobs.

Reply:
The way the tests are written require both a github org and moment so let's not change it right now